src/locales/ko/move-trigger.ts

```diff
@@ -33,6 +33,7 @@ export const moveTriggers: SimpleTranslationEntries = {
   "isChargingPower": "{{pokemonName}}는(은)\n파워를 모으고 있다!",
   "burnedItselfOut": "{{pokemonName}}의 불꽃은 다 타 버렸다!",
   "startedHeatingUpBeak": "{{pokemonName}}는(은)\n부리를 가열하기 시작했다!",
+  "setUpShellTrap": "{{pokemonName}} set up a shell trap!",
```

Comment:
```suggestion
  "setUpShellTrap": "{{pokemonName}}는(은)\n트랩셸을 설치했다!",
```
<img width="1641" alt="Screenshot 2024-08-12 at 10 04 53 AM" src="https://github.com/user-attachments/assets/aa607233-0fe4-4871-ab2f-3fd9f6c3b755">

Reply:
@KimJeongSun
This can be replaced 는(은) to [[는]]

Note : i18next가 조사를 자동으로 세팅하게끔 [[는]]을 붙이는게 맞습니다.
현재 '는(은)'을 붙인 문장들은 i18next를 호출한 후 조사 앞 단어를 replace로 후처리하는 케이스라 임시로 처리한 경우입니다.

Reply:
can we replace every `는(은)` to `[[는]]` if it possible?